website/pages/docs/rules.mdx

```diff
@@ -25,9 +25,9 @@ You are probably here because you got a warning message in your console.
 
 There are three common reasons you might be seeing it:
 
-1. You might be **breaking Rules of Blocks.**
-2. You might have **forgot to use the compiler.**
-3. You've entered **unsupported behavior.**
+1. Breaking [rules of blocks](/docs/rules#breaking-rules-of-blocks)
+2. Not [adding the compiler](/docs/rules#using-the-compiler)
+3. Encountering [unsupported behavior](/docs/rules#unsupported-behavior)
 
 ## Breaking Rules of Blocks
```

Comment:
Some comments regarding this page:
- The Error messages are probably better suited for code blocks vs asides. This makes them look (imo) out of place, whereas a snippet of what the error message looks like in the console could be better.
A pattern you can follow:
```md
When you have a block that is not declared as a variable, you may encounter an error message (or warning?):

`[Million.js] Block needs to be defined as a variable declaration.`

This message tells you that the compiler can not analyze the block correctly and ... [explain examples of why]

[insert examples of errors & corrections here]
```
   - I tend to follow this example vs telling someone to "look above" because I want a page to have flow. Meaning you read it top to bottom. Now this is more personal preference, but I find having moving up and down the page to get context can be disorientating 
   - There's still some emojis on this page outside of code blocks I would remove. Using ❌ and ✅  are nice to emphasize points in code where italics and bolds can't be used